.circleci/config.yml

```diff
@@ -412,8 +435,52 @@ jobs:
                       --signer-logging-level="INFO"\
                       --attestor-key-id="//cloudkms.googleapis.com/v1/projects/$ATTESTOR_PROJECT_NAME/locations/global/keyRings/$ATTESTOR_NAME-key-ring/cryptoKeys/$ATTESTOR_NAME-key/cryptoKeyVersions/1"
 
+  # Verify newly published images (built on ARM machine) will run on AMD
+  check-cross-platform:
+    #machine:
+      #image: ubuntu-2204:2023.07.1
+    docker:
+      - image: cimg/base:current
+    resource_class: arm.medium
```

Comment:
The `resource_class` here seems correct, but in the comments and the check it appears like it's checking for AMD support, but it should be checking for ARM support, as it's build and tested on AMD during regular CI flow already.

Reply:
Ugggh that's my bad. Good catch. Getting myself confused between the two 🤦‍♂️ . Will change the comments and the initial `Verify Image Platform` check to look for `arm64` in the docker manifest.

Reply:
It does appear that when testing the new Circle CI `check-cross-platform` job worked, at least when pulling the ARM version of the `op-node:e87e5ef2b96893eb8b446da420f7ba7f3e3c5985` container and checking the version. [Here](https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/53233/workflows/dc51b6a8-c3b5-4bf0-af5c-113f9a73abb6/jobs/2305862) is that run.

I went ahead and deleted the test invocation, and added triggers for the new job after we publish each of our docker containers, but let me know if you think it should be triggered differently